CMakeLists.txt

```diff
@@ -35,7 +35,8 @@ option(BUILD_FILTERS "Build filter applications." ON)
 option(BUILD_GENERATORS "Build code generation applications." ON)
 
 if(WIN32)
-    set(SHELL PowerShell -Command)
+    # Use bash (usually from Git for Windows) for piping results
+    set(SHELL bash -c)
```

Comment:
Do we know of any Windows users that can comment on this?

I know that I have bash on my Windows VM, but is that true for those who treat it as a daily driver and use git?